app/models/organization_capacity.rb

```diff
@@ -0,0 +1,18 @@
+# == Schema Information
+#
+# Table name: organization_capacities
+#
+#  active_client_count :bigint
+#  capacity_limit      :integer
+#  name                :string
+#  vita_partner_id     :bigint
+#
+class OrganizationCapacity < ApplicationRecord
+  self.primary_key = :vita_partner_id # vita partner id
```

Comment:
Consider renaming the column in the VIEW so it's `id` and then we can ditch this line. (Then again, if you add the `belongs_to` below, it's actually nice that this is `vita_partner_id`.)

Reply:
Consider adding Rails `belongs_to :vita_partner` so you can do `OrganizationCapacity.where(vita_partner: { name: "Oregon Helpers" })` or whatever.

Reply:
Perhaps the inverse, `has_one :organization_capacity` might be cool on `class VitaPartner`.

Reply:
done!